app/views/contexts/section_summary_context.py

```diff
@@ -86,7 +86,7 @@ def get_page_title(self, title_for_location: str) -> str:
             page_title = page_title.format(list_item_position=list_item_position)
         return page_title
 
-    def _build_summary(self, return_to: Optional[str]):
+    def _build_summary(self, return_to: Optional[str]) -> Any:
```

Comment:
This should return a dict

Reply:
Looked again... it returns `Union[dict[str, list[dict[str, Any]]], dict[str, list[Mapping[str, Any]]]]`

Reply:
The values of the dict aren't always a list. It can be simplified to `dict[str, Any]` as it can't be typed more accurately without a data model or TypedDict.

Reply:
I already updated it to dict[str, Any] with the latest change.